src/fake-timers-src.js

```diff
@@ -421,8 +421,12 @@ function withGlobal(_global) {
         if (typeof timer.func === "function") {
             timer.func.apply(null, timer.args);
         } else {
-            /* eslint no-eval: "off" */
-            eval(timer.func);
+            throw new Error(
+                "TypeError [ERR_INVALID_CALLBACK]: Callback must be a function. Received " +
```

Comment:
Can you change the `Error` to be a `TypeError` and remove the "TypeError" prefix from the message? This would currently serialize as `Error: TypeError …`.

It would also help with readability of the message if the actual received value would be quoted.

Reply:
Delete the 'TypeError' and let it start with [ERR_INVALID_CALLBACK]?

Reply:
Yes. And throw a `TypeError` instead of an `Error`.

Reply:
Ohhh ok! Now I get it :) I never used a different type of error before, thanks for teaching me something new!